admin_manual/configuration/server/occ_command.rst

```diff
@@ -1502,6 +1502,17 @@ Shibboleth
 ::
 
   sudo -u www-data ./occ user:sync "OCA\User_Shibboleth\UserBackend"
+
+Syncing via cron job
+~~~~~~~~~~~~~~~~~~~~
+
+::
```

Comment:
If I'm reading it correctly, this should come before `crontab -e`. Otherwise, it looks good.